### DISCUSSION TITLE
"This project is not being actively developed at the moment."

### DISCUSSION THREAD

**Discussion by JupiLogy** [lebrice/Sequoia#264] [General]
I was just wondering... This seems like quite a nice and useful package. Though it seems that it's run by, pretty much, just @lebrice . 

It makes sense to not be able to continue developing a large package, solo, without it being literally your job. Though I was wondering, do people use this package? Or are there similar packages that are under active development that are more popular? Will this project be more active again in future?

Just wondering as I'm hoping to add some environments/task schedules. At least I'm planning to add them here anyway. But now I've found this package I may use it in future projects, if it is functional enough.

**Comment by lebrice** [comment]
Hey there @JupiLogy , sorry for the late reply!

I'd love to be able to keep working on this, but yeah I've been working on other projects lately as part of my new job, and I have not yet been assigned time to work on this one.

To be perfectly honest, I don't know if anyone else is using this. I haven't heard of any particularly similar libraries, apart from maybe Avalanche, which is a bit different, but seemed to have a bigger crowd.
I hope that I can work on Sequoia again in the future, but for now I can't make any promises. 
You're more than welcome to add some environments and task schedules if you'd like, and Sequoia is functional enough in my view to be used in your projects if you'd like. If you were to encounter some issues, I'd be happy to help you out.

Thanks for posting this though, it really makes me feel good to know that someone else also thinks this is useful! :)
Let me know if you decide to use it, and if you need some help adding your envs / tasks I'd be happy to assist you in whatever way I can.

**Reply by JupiLogy** [reply]
Hey @lebrice no worries :)

Although the main avalanche repo is actively developed and has many stars, the avalanche-rl repo by the same folks (continualAI) was last updated over a year ago and has less stars than this repo. And the main avalanche repo does not have support for reinforcement learning. So for me, at least, sequoia has an edge!

I've ended up continuing to use my own hodge-podge-jank setup for now, since I have a thesis deadline to meet....... But it would be a lot nicer to be using something organised like this! So if I decide I have to tidy everything up, this will be up there on the list of "templates/libraries" or whatever this might be called. ✨
